Python/PRP/Surface/pyCubicRenderTarget.cpp

```diff
@@ -0,0 +1,156 @@
+/* This file is part of HSPlasma.
+ *
+ * HSPlasma is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * HSPlasma is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <PyPlasma.h>
+#include <PRP/Surface/plRenderTarget.h>
+#include "pyRenderTarget.h"
+#include "PRP/pyCreatable.h"
+
+static inline plCubicRenderTarget* IConvertRT(pyCubicRenderTarget* self) {
+    return plCubicRenderTarget::Convert(IConvert((pyCreatable*)self));
+}
+
+extern "C" {
+
+static PyObject* pyCubicRenderTarget_new(PyTypeObject* type, PyObject* args, PyObject* kwds) {
+    pyCubicRenderTarget* self = (pyCubicRenderTarget*)type->tp_alloc(type, 0);
+    if (self != NULL) {
+        self->fThis = new plCubicRenderTarget();
+        self->fPyOwned = true;
+    }
+    return (PyObject*)self;
+}
+
+static PyObject* pyCubicRenderTarget_getLeftFace(pyCubicRenderTarget* self, void*) {
+    return pyRenderTarget_FromRenderTarget(IConvertRT(self)->getFace(plCubicRenderTarget::Faces::kLeftFace));
```

Comment:
Non-standard C++:  `enum Faces` is not an `enum class`, so these should be `plCubicRenderTarget::kLeftFace`, etc.